### PR TITLE
add draw.io high-level design

### PR DESCRIPTION
**Pull request by petrolifero**
The design of a initial aws infrastructure, without vpc and firewall for simplicity. This diagram will be show to all team on 05/15/2021.

### CODE REVIEW

File: Infra-Design.drawio [hunk 1]
```diff
@@ -0,0 +1 @@
...JuG05RgrLgNzRFpJhc++9lqpx69k6Ygwd3Xg3YMw7PCg07Sb81X+4cvoSBnwA+jJNiKr0ZV46fVn19HU5C7jQm498b8vyzTzq8s10t/BEef1m6dZvLtqu7r1Wb8579TyHi4ON87koBCTAoLbLLeRKvg7KXR3IRykrRGTsHBlCzq76x3FqbkOnFj57iyv2aoIcrtOrCdu7ubW6XUlDMnn77G4NNMg/VM2ix8bcz/8xE4H9VNKFWCPUyWxVCNRq7D+ge4Is1MlsVQjUamlJjLoq1MlsrI5Ybg00rYHUOuOUvdI5KVBtZ2iUgToIItZRbi2LNPBQwT0wcB/YunBhkv2cwBj/OOmjTfAwys48+cM4HU/K4PlVtRZ5ig9jsRjDKhkXAUtdDsg5lsHCNx6IknWQ5FkEzEs/Snc2CYS08FoqHC590DigRWcMaKUV0J3ldqFR+pGyVI13YKAc3zIXAI8T4FrSg+zdAS50jP3qAwccPyIGui1I7T71lcfEBVEdICgGUIISvPiYuEG29xZX3eKqK42rkCVxd3FI62xx1Zvam438WAUjHNqo5xx1BWtjWW+GqeoNqnpzdpDzfnrT5V3eUjzcPo4VPqYcyM7xt88PX9B3r/e7N/71E36agfXdGeNYCyhBWsudWbu2p2OfULjeFIDYJbpopJmmhA8TtESaISPNdeVTgcfGmm7j/eaNZG8EzK60MjAtTRxxSn8k4phrZAmjIUu452MJJevW9iiLci6z4VGWQzFEYS3XhzOrIcycM8LMsaRtNgQt2Yc0hpq0F1g6j3V4sGlnsu2HVM7ijeL/NN5o4H6wTVDnjTR7VQpAd3gjWGUErHFFR8vuafUG6/X2BjhCMuBa0tgF4fpNf/d0JCFtgqC2R7fNuo6OTA8N9jpv9CCffb1jbCDI4HwM8T4+3nEQhmi6m4LPRhB3bfdOZIJQOjoyQTQ4tn85BLHfAaG9OMKS9YQMdG6GaLAvfi0MgZvGENbJKMIC8kLjDtgt970sW/ZQprJmOTJRNNhivWVNb1nTt5s13c89GLKxA2EgJ0ib7trKuxnszWBvBqt+jsmu5qSQpQnnTmmuTY7qX0s05zRc750whanwO4LK95E0juZMubNiQ/JUu8bvI0VxELC5Fwc2rKQoIG6dorCgEpnIGdbWYGPFzXd05dU3X3WGhv8D</diagram></mxfile>
```
Comment (petrolifero) [inline Infra-Design.drawio]: Where to test this file?

Comment (petrolifero) [reply Infra-Design.drawio]: Save this file on computer.
On https://app.diagrams.net/  , using File -> Import From -> Device
Select this file and the diagram will be showed

**Comment by dbofmmbt** [comment]
@petrolifero could you please add something like labels to the document? I'm not familiar with some of the icons.

**Review by camilletantow — APPROVED** [review]

**Review by dbofmmbt — APPROVED** [review]
The high-level design looks really good! The only thought I had was that we may not need SQS in the beginning, but it will certainly be great when we perform load tests on the system.

**Comment by petrolifero** [comment]
> 
> 
> The high-level design looks really good! The only thought I had was that we may not need SQS in the beginning, but it will certainly be great when we perform load tests on the system.

I agree

**Review by dbofmmbt — APPROVED** [review]

**Review by camilletantow — APPROVED** [review]

**Pull request merged**